paper.md

```diff
@@ -72,10 +76,10 @@ HPX is currently funded by
   any information, apparatus, product, or process disclosed, or represents that
   its use would not infringe privately owned rights.
 
-* The Bavarian Research Foundation (Bayerische Forschungsstiftung) through the
+- The Bavarian Research Foundation (Bayerische Forschungsstiftung) through the
   grant AZ-987-11.
 
-* The European Commission's Horizon 2020 programme through the grant
+- The European Commission's Horizon 2020 programme through the grant
```

Comment:
I guess the two above are not active anymore. So hpx is technically not currently funded by those.

Reply:
We might want to rephrase this by saying 'has been funded'.

Reply:
Like that